### ISSUE TITLE
Adding some content to the repo

### ISSUE THREAD

**Issue by jlasoc1** [ddd-crew/ddd-starter-modelling-process#34]
Hello everyone, I'm Javiera Laso and I was one of the speakers at the ddd eu 2022. Talking with @yellowbrickc she ask us to contribute in some way to this repository, but I'm not sure how to do it yet. Here are my slides: [How to start with ddd when you have a monolith](https://speakerdeck.com/javujavichi/how-to-start-with-ddd-when-you-have-a-monolith), I haven't written any post related to this talk (because I just finished the presentation about a month before the conference), but there are some interesting things that can be explained in detail. 
So my question is, how can I help?

**Comment by yellowbrickc** [comment]
I think we can always have more content helping people start with DDD. I haven't thought of the slides but rather an article, a document about the message of your talk @jlasoc1 if you find the time to write it. 

What do you think @ddd-crew/crew ? (Unfortunately, I haven't seen any of the talks on the foundation day but I am sure there is a lot that would complete our current collection)

**Comment by NTCoding** [comment]
@yellowbrickc I think you make a good point. Some documents and case studies would be good, or also an example of going through all of these steps.

I created a couple of Katas using Miro that touch on some of the steps, but not something that goes all the way through and is linked in this repo: https://medium.com/nick-tune-tech-strategy-blog/architecture-ddd-kata-online-car-dealership-540c534121e2

**Comment by Max-Git** [comment]
During my DDD Foundations' talk about the Starter Modelling Process, I showed stuff I've done with my clients for almost every steps. I could upload some of them on Github. 
Would those examples be embeded in the main README file or in a separate markdown?

**Comment by emgsilva** [comment]
In the README in-line may turn this into a huge/long thing to go through...

I would say either: 1) different pages under a new "usecases" directory,
which we link from Readme. or, 2) use a "collapsible" section (something
like
https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab).

Somehow I think 1) would be a better design as we can refer to those use
cases on different locations and even in isolation... Plus we can set a
nice template to collect use cases.

**Comment by NTCoding** [comment]
I agree with the comments above. I don't like agreeing with people, but this is too good not to agree with.

I think a separate folder for examples, and possibly even a template, could make it easy for other people to contribute. Just drop a markdown file in the examples folder and raise a PR.

**Comment by emgsilva** [comment]
It shows a lot of maturity when one puts the good of the whole over going
for the default (disagreeing) behavior 😜

**Comment by yellowbrickc** [comment]
Sorry that I was so silent, but now I am here 😁 
In https://github.com/ddd-crew/aggregate-design-canvas we have a directory called "examples", in the BC-Canvas-Repo we have "tools". I propose having a directory "case-studies" and referencing it in the Readme when the first study is checked in.

Thanks, @Max-Git beforehand!

**Comment by emgsilva** [comment]
+1 for `case-studies` (maybe we can adopt that as the default on all the repos - and try to get more of them in all of them).

**Comment by NTCoding** [comment]
@emgsilva are you doing this on purpose to make me keep agreeing with you?

**Comment by emgsilva** [comment]
trying to replicate the situation and validate if we can make this a "real pattern" for the books 📖 😁

**Comment by yellowbrickc** [comment]
Happy New Year everyone 🎉 

The directory and a minimal How-to are added. Now we only need the case studies 😁
